src/rest/Query.ts

```diff
@@ -25,6 +25,16 @@ export interface IUserActionsRequest {
   tagViewsOfUser: string;
 }
 
+/**
+ * The information about an e-commerce search request.
+ */
+export interface ICommerceRequest {
+  /**
+   * The id of the catalog we would like to query.
```

Comment:
Suggestion:

```
The unique identifier of the catalog to query.

**Example:** `46bc4275-e613-4dd5-b1ea-3e5aca1bcd9d`
```

(I'm assuming catalog IDs are generated as GUIDs; if this is not the case, please replease the example I suggest by something that makes sense :) ).